client/snippets/snippets.json

```diff
@@ -39,5 +52,267 @@
+            "\t$0",
+            "\\endStyle"
+        ],
+        "description": "Styled text"
+    },
+    "Style (custom)": {
+        "prefix": "\\beginStyle",
+        "body": [
+            "\\beginStyle(-> Style",
+            "\t--> ${1:attrs}",
+            ";)",
+            "\t$0",
+            "\\endStyle"
+        ],
+        "description": "Text using a custom style"
+    },
+    "Bold Text": {
+        "prefix": "bold",
+        "body": [
+            "\\beginStyle(BOLD_STYLE)",
+            "\t$0",
+            "\\endStyle"
+        ],
+        "description": "Bold text"
+    },
+    "Italic Text": {
+        "prefix": "italic",
+        "body": [
+            "\\beginStyle(ITALIC_STYLE)",
+            "\t$0",
+            "\\endStyle"
+        ],
+        "description": "Italic text"
+    },
+    "Quoted Text": {
+        "prefix": "\\beginQuote",
+        "body": [
+            "\\beginQuote(_${1|DOUBLE_QUOTE,RAW_DOUBLE_QUOTE,RAW_SINGLE_QUOTE,DOUBLE_QUOTE_EN,DOUBLE_QUOTE_FR,DOUBLE_QUOTE_DE,DOUBLE_QUOTE_JA,DOUBLE_QUOTE_TITLE_JA,SINGLE_QUOTE|})",
+            "\t$0",
+            "\\endQuote"
+        ],
+        "description": "Text placed between quotes"
+    },
+    "Complete": {
+        "prefix": "complete",
+        "body": [
+            "complete ${1:name}",
+            "--> $0",
+            ";"
+        ],
+        "description": "Complete an instance with values"
+    },
+    "Recipient Action": {
+        "prefix": "\\recipientAction",
+        "body": "\\recipientAction(VERB_${1|EN,FR,DE,ES,JA,NL,PT|}_${2:VERBNAME}, ${3|PRESENT_INDICATIVE,AUX_MOD,AUX_MOD_CONTINUOUS,AUX_MOD_PAST_PERFECT,AUX_MOD_PAST_PERFECT_CONTINUOUS,AUX_MOD_PERFECT,AUX_MOD_PERFECT_CONTINUOUS,AUX_MOD_PRETERIT,AUX_MOD_PRETERIT_CONTINUOUS,CONDITIONAL,CONJUNCTIVE_ROOT,CONTINUOUS_FUTURE,CONTINUOUS_FUTURE_PERFECT,CONTINUOUS_PAST_CONDITIONAL,CONTINUOUS_PAST_PERFECT,CONTINUOUS_PRESENT,CONTINUOUS_PRESENT_CONDITIONAL,CONTINUOUS_PRESENT_PERFECT,CONTINUOUS_PRETERIT,FUTURE,FUTURE_1_INDICATIVE,FUTURE_2_INDICATIVE,FUTURE_INDICATIVE,FUTURE_PERFECT,FUTURE_PERFECT_INDICATIVE,FUTURE_PERFECT_SUBJUNCTIVE,FUTURE_SUBJUNCTIVE,FUTURE_SUBJUNCTIVE_1,FUTURE_SUBJUNCTIVE_2,GERUNDIVE,IMPERATIVE,IMPERFECT_INDICATIVE,IMPERFECT_SUBJUNCTIVE,INACCOMPLISHED_ROOT,INFINITIVE,INFINITIVE_1,INFINITIVE_2,PARTICIPLE,PARTICIPLE_1,PARTICIPLE_2,PARTICIPLE_3,PAST,PAST_CONDITIONAL,PAST_CONDITIONAL_SECOND_FORM,PAST_GERUNDIVE,PAST_HISTORIC_INDICATIVE,PAST_IMPERATIVE,PAST_INFINITIVE,PAST_PARTICIPLE,PAST_PERFECT,PAST_PERFECT_INDICATIVE,PAST_SUBJUNCTIVE,PAST_SUBJUNCTIVE_1,PAST_SUBJUNCTIVE_2,PERFECT_INDICATIVE,PLU_PERFECT_INDICATIVE,PLU_PERFECT_SUBJUNCTIVE,POTENTIAL,PRESENT,PRESENT_CONDITIONAL,PRESENT_IMPERATIVE,PRESENT_INFINITIVE,PRESENT_OR_FUTURE,PRESENT_PARTICIPLE,PRESENT_PERFECT,PRESENT_PERFECT_INDICATIVE,PRESENT_SUBJONCTIVE_1,PRESENT_SUBJONCTIVE_2,PRESENT_SUBJUNCTIVE,PRETERIT,PRETERIT_INDICATIVE,TE_FORM,VOLITIONAL|}_$1)",
+        "description": "Recipient verb"
```

Comment:
I don't think anybody can understand what is a “recipient verb”.

Reply:
That's why I was asking for your help regarding descriptions and names xD

Reply:
What about “Second person subject verb”?